module/local/me_test.go

```diff
@@ -0,0 +1,30 @@
+package local
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/onflow/flow-go/utils/unittest"
+)
+
+func TestInitializeWithMatchingKey(t *testing.T) {
```

Comment:
I would like to see at least a minimal godoc for each test. Just a quick summary.